tests/figures/chart/__snapshots__/chart_plugin.test.ts.snap

```diff
@@ -228,6 +259,33 @@ exports[`Linear/Time charts snapshot test of chartJS configuration for date char
     "plugins": [],
     "type": "line",
   },
+  "dataSetFormat": undefined,
+  "dataSetsValues": [
```

Comment:
What's up with the snapshots ? Why do theses snapshots suddenly have datasets values that weren't there before when you didn't change the tests ? And why does `chart_component.test.ts` suddenly have snapshots, when there is zero snapshot tests in the file ?

Might be what I was talking about when I said taht you shouldn't import things from a test file to another 🙃

Reply:
Did you find why snapshots are so different now ?

Reply:
It comes from the removal of the `createLineChartRuntime` method :)